CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/GenericJsonIO/JsonIO.java

```diff
@@ -0,0 +1,235 @@
+/*
+ * Copyright 2010-2019 Australian Signals Directorate
```

Comment:
Can you please update your NetBeans settings to 2020?

Reply:
Sure, no worries.